flow360/examples/actuatorDisk/flow360.json

```diff
@@ -38,10 +38,6 @@
       "animationFrequency": -1,
       "nodeForcesPerUnitArea": true
     },
-    "sliceOutput": {
-      "outputFormat": "paraview",
-      "animationFrequency": -1
-    },
```

Comment:
Why is this deleted? To avoid user copying and using senseless JSON?

Reply:
I remember this actually serves as a good test for updater so it handles all situations. See the PR I merged yesterday for numericalDissipation.

Reply:
ok